Script/AtomicEditor/ui/frames/MainFrame.ts

```diff
@@ -111,12 +111,18 @@ class MainFrame extends ScriptWidget {
         if (this.menu.handlePopupMenu(target, refid))
             return true;
 
+        if (this.openedMenu && this.openedMenu.id == target.id + " popup") {
+            this.openedMenu.close();
+            this.openedMenu = null;
+            return true;
+        }
+
```

Comment:
What happens when the menu is closed normally, by selecting something in it or clicking off it?  Does the openMenu member still hold a reference to the last UIMenuWindow?  It seems like it would